jmix-flowui/flowui/src/main/java/io/jmix/flowui/data/value/ContainerValueSource.java

```diff
@@ -372,4 +376,17 @@ protected boolean isEqualCollectionValue(@Nullable V a) {
         //noinspection ConstantConditions
         return CollectionUtils.isEqualCollection(newValue, oldValue);
     }
+
+    protected void setCollectionValue(E item, @Nullable V value) {
```

Comment:
Before this change, MultiValuePicker and similar components could not be bound to a Set attribute.